test/lib/tpunit++.cpp

```diff
@@ -396,6 +403,27 @@ void tpunit::TestFixture::tpunit_detail_do_tests(TestFixture* f) {
     }
 }
 
+void tpunit::TestFixture::testLog(const string& newLog) {
```

Comment:
Same here, I don't think we need both of these functions, just this one (but with the lock that's in the one below)